.github/workflows/dependencies/dpcpp.sh

```diff
@@ -29,16 +29,16 @@ df -h
 # https://github.com/BLAST-WarpX/warpx/pull/1566#issuecomment-790934878
 
 # try apt install up to five times, to avoid connection splits
-# FIXME install latest version of IntelLLVM, Intel MKL
-#       after conflicts with openPMD are resolved
```

Comment:
@ax3l I updated the code that uses MKL FFT in this PR. What were the conflicts with openPMD?

Reply:
Okay it was added in #5419. Since there is no error now, it is safe to remove it.